docs/_docs/windows.md

```diff
@@ -45,6 +45,7 @@ sudo gem install jekyll bundler
 You can test by running:
 
 ```
+cd ~
```

Comment:
New users (as I have seeing them) want to perform the steps of some manual to get a just working environment. A professional one will pass `cd ~` as a known step (or choose the directory as he prefers). But a novice will pass that comment text just going step by step in the `code` frames only. Of course the good knowledge of the English language is also an expected prerequisite for you. Please sorry for mine.